projects/plugins/jetpack/extensions/blocks/amazon/amazon.php

```diff
@@ -12,8 +12,14 @@
 use Automattic\Jetpack\Blocks;
 use Jetpack_Gutenberg;
 
-const FEATURE_NAME = 'amazon';
-const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+/**
+ * Returns the path to the directory containing the block.json file.
+ *
+ * @return string
+ */
+function get_block_path() {
+	return dirname( JETPACK__PLUGIN_FILE ) . '/_inc/blocks/amazon';
+}
```

Comment:
Maybe this is something that could be extracted to the Blocks package, since I foresee us doing this for other blocks later?